ui/app/services/capabilities.ts

```diff
@@ -106,12 +110,18 @@ export default class CapabilitiesService extends Service {
     }
   }
 
+  // convenience method for fetching capabilities for a singular path without needing to use pathFor
+  async for<T>(key: keyof typeof PATH_MAP, params?: T) {
```

Comment:
My only concern with the naming is the overlap with the javascript `for` method. When I later see it applied (ex: `capabilities.for(....)`) my mind jumps to the native js method. Could we rename this to something more descriptive? 
Maybe: `forPath`

Reply:
I think `forPath` is a bit confusing since there is already a `pathFor` method on the service. I guess I was thinking along the lines of `modelFor` in Ember and since the service is called `capabilities` it would have been odd to call it `capabilitiesFor` so I settled on simply `this.capabilities.for()`.